mapdata/WarcraftLegacies/UpgradeData/ZBML.json

```
@@ -102,7 +102,7 @@
       "Level": 1,
       "Id": 828536167,
       "Type": 3,
-      "Value": "Allows Obsidian Eradicators to use Spell Conduction, which causes 35% of Magic and Spell damage dealt to friendly units within a radius of 500 to be redirected to them. The damage is affected by the resistances of both the damaged unit and the Obsidian Eradicator. Also grants 100 extra Hit Points |n|n|cffc864c8Can\u0027t be researched with Shaped Obsidian.|r"
+      "Value": "Allows Obsidian Eradicators to use Spell Conduction, which causes 40% of Magic and Spell damage dealt to friendly units within a radius of 500 to be redirected to them. Damage from other Spell Conduction units is not redirected. The damage is affected by the resistances of both the damaged unit and the Obsidian Eradicator. Also grants 100 extra Hit Points.|n|n|cffc864c8Can't be researched with Shaped Obsidian.|r"
```

I think it's not necessary to tell the player that Spell Conduction doesn't bounce; logically it can't, since that would be an infinite loop.
```suggestion
      "Value": "Allows Obsidian Eradicators to use Spell Conduction, which causes 40% of Magic and Spell damage dealt to friendly units within a radius of 500 to be redirected to them. The damage is affected by the resistances of both the damaged unit and the Obsidian Eradicator.|n|n|cffc864c8Can't be researched with Shaped Obsidian.|r"
```